src/stock.rs

```diff
@@ -356,22 +341,18 @@ mod tests {
                 DC_CONTACT_ID_SELF as u32
             ),
             "Member Alice (alice@example.com) added by me."
-        )
+        );
     }
 
     #[test]
     fn test_stock_system_msg_add_member_by_other_with_displayname() {
         let t = dummy_context();
-        let contact_id = unsafe {
-            let name = CString::new("Alice").unwrap();
-            let addr = CString::new("alice@example.com").unwrap();
+        let contact_id = {
             assert!(
-                dc_create_contact(&t.ctx, name.as_ptr(), addr.as_ptr()) > 0,
+                Contact::create(&t.ctx, "Alice", "alice@example.com").unwrap() > 0,
                 "Failed to create contact Alice"
             );
-            let name = CString::new("Bob").unwrap();
-            let addr = CString::new("bob@example.com").unwrap();
-            let id = dc_create_contact(&t.ctx, name.as_ptr(), addr.as_ptr());
+            let id = Contact::create(&t.ctx, "Bob", "bob@example.com").unwrap();
             assert!(id > 0, "Failed to create contact Bob");
```

Comment:
This assert can also be removed as the `.unwrap()` on the previous line does that already (or use `.expect()` I guess, but really the original error should be fine.